### PR TITLE
Projects with JSON Files 

### PR DESCRIPTION
**Pull request by haydavid23**

### CODE REVIEW

**Comment by alesanchezr** [comment]
@haydavid23 why is it giving conflicts? Did you modify the info.json files?

**Comment by haydavid23** [comment]
@alesanchezr I created a new folder ("projects") inside the "P" folder with all of the project as JSON. I noticed that all of the conflicting files path are incorrect. the path for career-support should be : p/**CSS/**career-support/xxx/xxx.json AND  the path for final-project should be : p/final-project/**full-stack**/web-development/wireframes/info.json. 

was the project modified after I made the fork???

**Comment by haydavid23** [comment]
@alesanchezr . I dont know what happened, but it appears that I changed the location of career support and final project. I will fix it the issue. give me a sec

**Comment by haydavid23** [comment]
. try again .. i fixed it

File: p/test.js [hunk 1]
```diff
@@ -35,23 +35,41 @@ https.get(url, res => {
 
 
 let readmeArray = [];
-let technologies = ['css','javascript','php','jquery','python','data-modeling']
+let technologies = ['css','javascript','php','jquery','python','data-modeling', 'final-project']
+let level = ["beginner", "junior", "senior","semi-senior", "full-stack"]
+let projects = []
+
+fs.readdirSync("./css/beginner").forEach(e=>projects.push(String(e)))
```
Comment (alesanchezr) [inline p/test.js:42]: Porque estas buscando en carpetas en particular? estos nombres no pueden estar hardcoded.

Tenemos que sentarnos porque el algoritmo no esta haciendo exactamernte lo que que yo queria.

**Review by alesanchezr — CHANGES_REQUESTED** [review]

File: p/test.js (hunk 1 above)
Comment (haydavid23) [reply p/test.js:42]: :(   ok.....  yo manana y el viernes voy a estar todo el dia en la academia.

**Pull request closed without merging**